web/src/cypress/support/service.ts

```diff
@@ -221,6 +221,7 @@ function createIntKey(
       'grafana',
       'site24x7',
       'prometheusAlertmanager',
+      'universal',
```

Comment:
we can leave this out at this stage. UIK will get it's own test entirely since it's a very different flow & use case

Reply:
https://github.com/target/goalert/pull/3837